apps/dh/app-dh/src/main.ts

```diff
@@ -15,29 +15,37 @@
  * limitations under the License.
  */
 import { enableProdMode } from '@angular/core';
-import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+import { bootstrapApplication } from '@angular/platform-browser';
+import { provideRouter } from '@angular/router';
 import {
   dhApiEnvironmentToken,
   dhB2CEnvironmentToken,
   dhAppEnvironmentToken,
   environment,
 } from '@energinet-datahub/dh/shared/environments';
 
-import { DataHubAppModule } from './app/datahub-app.module';
+import { DataHubAppComponent } from './app/datahub-app.component';
+import { DhCoreShellProviders, DhCoreShellRoutes } from '@energinet-datahub/dh/core/shell';
 import { loadDhApiEnvironment } from './configuration/load-dh-api-environment';
 import { loadDhB2CEnvironment } from './configuration/load-dh-b2c-environment';
 import { loadDhAppEnvironment } from './configuration/load-dh-app-environment';
+import { provideAnimations } from '@angular/platform-browser/animations';
 
 if (environment.production) {
   enableProdMode();
 }
 
 Promise.all([loadDhApiEnvironment(), loadDhB2CEnvironment(), loadDhAppEnvironment()])
   .then(([dhApiEnvironment, dhB2CEnvironment, dhAppEnvironment]) => {
-    platformBrowserDynamic([
-      { provide: dhApiEnvironmentToken, useValue: dhApiEnvironment },
-      { provide: dhB2CEnvironmentToken, useValue: dhB2CEnvironment },
-      { provide: dhAppEnvironmentToken, useValue: dhAppEnvironment },
-    ]).bootstrapModule(DataHubAppModule);
+    bootstrapApplication(DataHubAppComponent, {
+      providers: [
+        { provide: dhApiEnvironmentToken, useValue: dhApiEnvironment },
+        { provide: dhB2CEnvironmentToken, useValue: dhB2CEnvironment },
+        { provide: dhAppEnvironmentToken, useValue: dhAppEnvironment },
+        provideRouter(DhCoreShellRoutes),
```

Comment:
suggestion: the config is missing in the new way of providing the Router. Take a loot at `provideRouter` [docs](https://angular.io/api/router/provideRouter) for guidance on how to configure the Router.